src/main/resources/standardTestCases/bad_docker_name.test

```diff
@@ -0,0 +1,7 @@
+name: bad_docker_name
+testFormat: workflowfailure
+
+files {
+  wdl: bad_docker_name/bad_docker_name.wdl
+  inputs: hello/hello.inputs
+}
```

Comment:
Yeah, at the moment it's a combined limitation of how centaur compares those values with what cromwell emits. Aside: Cromwell omits an extremely opaque error string in this case.